frontend/cypress/integration/featureFiles/services.feature

```diff
@@ -60,3 +60,34 @@ Feature: Kiali Services page
     And user filters for label "app=productpage"
     Then user sees "productpage" in the table
     And table length should be 1
+
+  @services-page
+  Scenario: The healthy status of a service is reported in the list of services
+    Given a service in the cluster with a healthy amount of traffic
+    When I fetch the list of services
+    And user selects the "bookinfo" namespace
+    Then the service should be listed as "healthy"
+
+  @services-page
+  Scenario: The idle status of a service is reported in the list of services
+    Given a service in the cluster with no traffic
+    When I fetch the list of services
+    And user selects the "default" namespace
```

Comment:
This will require updating the rest of the tests and not just the ones you've added but if we're testing things in namespaces outside of `bookinfo` we should probably move the `And user selects the "bookinfo" namespace` statement out of Background and into the individual tests.

Reply:
In general, I think it will be very hard to test a single Kiali page using the preset resources of only one namespace...

Reply:
And well... I'll move that statement outside the background.

Reply:
> In general, I think it will be very hard to test a single Kiali page using the preset resources of only one namespace...

That's probably true eventually but so far many of the tests have only relied on a single namespace. If all scenarios in a test require a single namespace to be selected then it's `Background`. Once the namespace differs between scenarios then it's no longer `Background`.

Reply:
It is moved.